platform/cervantes/disable-usbms.sh

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I imagine shellcheck should've already failed this while I was reviewing this.

```suggestion
#!/bin/sh

```

Reply:
Yeah, I didn't want to include the usbms because it isn't done yet. Now it is.